caesars-palace/blocks/header/header.js

```diff
@@ -129,6 +131,54 @@ function toggleMenu(nav, navSections, forceExpanded = null) {
   }
 }
 
+const showMore = (nav, maxItemsDesktop) => {
+  const ul = nav.querySelector('.nav-sections > .local-nav > ul');
+  const more = document.createElement('li');
+  more.classList.add('more');
+  const moreText = document.createElement('div');
+  moreText.classList.add('more-text');
+  const aMore = document.createElement('a');
+  aMore.classList.add('more-link');
+  aMore.text = 'MORE';
+  moreText.appendChild(aMore);
+  const dropdownMenu = document.createElement('div');
+  dropdownMenu.classList.add('dropdown-menu');
+
+  [...ul.children].forEach((child, index) => {
+    if (index < maxItemsDesktop) {
+      return;
+    }
+    const dropdownItem = document.createElement('div');
+    dropdownItem.classList.add('dropdown');
+    child.firstChild.classList.add('menu-item');
+    dropdownItem.appendChild(child.firstChild);
+    dropdownMenu.appendChild(dropdownItem);
```

Comment:
You can also replace this by having an array store the items and do only one append for all elements at once after the loop. Here, you do one DOM access in every loop iteration, you could have only one DOM access to add all elements created by the loop.